variables.tf

```diff
@@ -1,24 +1,79 @@
-/********************************************************************
+##############################################################################
+# Account Variables
+##############################################################################
 
-This file is used to declare ROOT varaibles.
+variable "install_storage" {
+  type        = bool
+  default     = true
+  description = "If set to false does not install storage and attach the volumes to the worker nodes. Enabled by default"
+}
+
+variable ibmcloud_api_key {
+  description = "Get the ibmcloud api key from https://cloud.ibm.com/iam/apikeys"
+  type        = string
+}
+
+variable unique_id {
+  description = "Unique identifiers for all created resources"
+  type        = string
+}
+
+variable cluster {
+  description = "Name of existing roks cluster"
+  type        = string
+}
+
+variable "kube_config_path" {
+  description = "Path to the k8s config file: ex `~/.kube/config`"
+  type        = string
+}
 
-E.g:
+variable resource_group {
+  description = "Resource group of existing cluster"
+  type        = string
+}
 
-variable "at_service_name" {
+##############################################################################
+
+##############################################################################
+# Block Storage Variables
+##############################################################################
+
+variable capacity {
+  description = "Capacity for all block storage volumes provisioned in gigabytes"
+  type        = number
+  default     = 100
+}
+
+variable profile {
+  description = "The profile to use for this volume."
```

Comment:
change the description of profile and capacity according to terraform reosures docs